src/D2L.Bmx/Aws/AwsCredsCache.cs

```diff
@@ -61,7 +61,7 @@ AwsCredentials credentials
 				&& o.Org == org
 			)
 			// Prune older (closer to expiry) credentials for the same role
-			.GroupBy( o => $"{o.AccountName}.${o.RoleName}", ( _, entries ) =>
+			.GroupBy( o => (o.AccountName, o.RoleName), ( _, entries ) =>
```

Comment:
Somewhat unrelated - this isn't a user-facing message, but it's also a string-related issue.

* [Value tuples](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples) are preferred when we want to use multiple values as a single key. Converting them to a single string is unnecessary and has more overhead.
* Even if we do want to use a string here, it should've been
  ```diff
  -$"{o.AccountName}.${o.RoleName}"
  +$"{o.AccountName}.{o.RoleName}"
  ```
  A common mistake when using C# interpolated strings.